bin/rails-engine

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+# This command will automatically be run when you run "rails" with Rails gems
+# installed from the root of your application.
+
+ENGINE_ROOT = File.expand_path('..', __dir__)
+ENGINE_PATH = File.expand_path('../lib/solidus_paypal_commerce_platform/engine', __dir__)
```

Comment:
Is it really what you intended, or included here by mistake?

Reply:
Thanks! fixed, also sorry, I forgot to set the PR back to draft after the CI turned out red for some ruby/rails/opal combo